.travis.yml

```diff
@@ -39,19 +39,13 @@ matrix:
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle and sevntu-checkstyle"
-        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dspotbugs.skip=true -Dcobertura.skip=true"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -DskipTests -DskipITs -Dpmd.skip=true -Dfindbugs.skip=true -Dspotbugs.skip=true -Djacoco.skip=true"
 
-    # jacoco (oraclejdk8)
+    # jacoco and codecov (oraclejdk8)
     - jdk: oraclejdk8
       env:
-        - DESC="jacoco"
-        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify -Pno-validations -Pjacoco-check -DskipTests=false"
-
-    # cobertura and codecov (oraclejdk8)
-    - jdk: oraclejdk8
-      env:
-        - DESC="cobertura and codecov"
-        - CMD="./.ci/travis/travis.sh cobertura-check"
+        - DESC="jacoco and codecov"
+        - CMD="export MAVEN_OPTS='-Xmx2000m' && mvn -e clean verify"
         - CMD_AFTER_SUCCESS="bash <(curl -s https://codecov.io/bash)"
```

Comment:
Verify phase will run all verification tools (checkstyle, pm, finbugs,.....) It will take time. But here we want to run only coverage check, please use '-Pno-validations....' as you did before. One day we will find way to run jacoco only.

Reply:
I've found the minimalistic version:
```
mvn clean test jacoco:restore-instrumented-classes jacoco:report@default-report
```